src/utils.ts

```diff
@@ -70,7 +72,14 @@ export function parseTags(content: string, firstLine = 0): Token[] {
 
     if (!content.startsWith(OPEN, pos)) continue;
 
-    const end = findTagEnd(content, pos) || 0;
```

Comment:
it felt odd to me to treat a non-found tag as position `0`. I think that breaks assumptions, so I thought an early continue would make more sense